docs/static/releasenotes.asciidoc

```diff
@@ -68,6 +69,82 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-16-0]]
+=== Logstash 8.16.0 Release Notes
+
+==== Announcing the new {ls} Health Report API
+
+The new Health Report API (`GET /_health_report`) is available starting with {ls} `8.16.0`.
+This API leverages indicators capable of detecting the degraded status of pipelines and
+providing actionable insights https://github.com/elastic/logstash/pull/16520[#16520], https://github.com/elastic/logstash/pull/16532[#16532].
+
+For a more detailed view please check <<logstash-health-report-api>>.
+
+[[featured-8.16.0]]
+==== New features and enhancements
+
+* {ls} now gracefully handles `if` conditionals in pipeline definitions that can't be evaluated (https://github.com/elastic/logstash/pull/16322[#16322]), either by dropping
+the event or by sending it to the pipeline's DLQ if enabled. https://github.com/elastic/logstash/pull/16423[#16423]
+
+* Added an escape hatch to prevent the new Health API's status from affecting the top-level `status` field of Logstash's existing API responses. By setting the `logstash.forceApiStatus` system property to `green` you can force the other APIs to return the previous hard-coded `green` value. https://github.com/elastic/logstash/pull/16535[#16535]
+
```

Comment:
A final suggestion is to move the force api status setting closer to the feature announcement to increase changes of users spotting it while reading the release notes. I rephrased it a little bit and added more context on why you'd need to tweak it:

```suggestion
The new Health Report API (`GET /_health_report`) is available starting with {ls} `8.16.0`.
This API leverages indicators capable of detecting the degraded status of pipelines and
providing actionable insights https://github.com/elastic/logstash/pull/16520[#16520], https://github.com/elastic/logstash/pull/16532[#16532].

NOTE for users upgrading to {ls} 8.16.0: if your existing automation relies on liveliness scripts that expect Logstash's API status to either not be available or return a hardcoded `green` status, there is an escape hatch to prevent the new Health API's status from affecting the top-level `status` field of Logstash's existing API responses. Setting the `logstash.forceApiStatus` system property to `green` will force other APIs to return the previous hard-coded `green` value. https://github.com/elastic/logstash/pull/16535[#16535]

For a more detailed view please check <<logstash-health-report-api>>.

[[featured-8.16.0]]
==== New features and enhancements

* {ls} now gracefully handles `if` conditionals in pipeline definitions that can't be evaluated (https://github.com/elastic/logstash/pull/16322[#16322]), either by dropping
the event or by sending it to the pipeline's DLQ if enabled. https://github.com/elastic/logstash/pull/16423[#16423]

```